.drone-v1.yml

```diff
@@ -0,0 +1,252 @@
+---
+
+kind: pipeline
+type: kubernetes
+name: build
+
+steps:
+- name: Build Project
```

Comment:
I don't like how these are proper nouns, but meh

Reply:
We can review these names at a later date.